package.json

```diff
@@ -5,7 +5,9 @@
     "start": "node ./bin/dungeon-revealer",
     "eslint": "eslint --ignore-path .gitignore --max-warnings 0 \"**/*.js\" \"bin/dungeon-revealer\"",
     "start:dev": "cross-env PORT=3001 EXTEND_ESLINT=true react-scripts start",
-    "build": "react-scripts build && pkg . --out-path ./bin/ --targets node10-win-x64,node10-macos-x64,node10-linux-x64",
+    "build": "react-scripts build",
+    "compilex64": "pkg . --out-path ./bin/ --targets node10-win-x64,node10-macos-x64,node10-linux-x64",
+    "compilearm": "pkg . --out-path ./bin/arm/ --targets node10-linux-armv7",
```

Comment:
@n1ru4l, are you fine with these changes? It changes when the binaries are built.
In order to build a binary on your system you would have to run: 
```
yarn install
yarn compilex64
```

Reply:
Yes fine with me